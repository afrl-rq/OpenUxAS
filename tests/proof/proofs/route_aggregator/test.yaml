filenames: ["route_aggregator.ads"]
level: 4
timeout: 120