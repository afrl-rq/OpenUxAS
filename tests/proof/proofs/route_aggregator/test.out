route_aggregator.adb:668:07: medium: exception might be raised
route_aggregator.ads:357:14: high: all paths in "Euclidean_Plan" raise exceptions or do not terminate normally
